iowrappers/photos_client.go

```diff
@@ -52,15 +56,15 @@ func (photoClient *PhotoHttpClient) GetPhotoURL(photoRef string) PhotoURL {
 		return photoURL
 	}
 
-	photoURL, err = parseHTML(body, isHtmlAnchor, "href")
+	photoURL, err = parseHTML(body, isHtmlAnchor, isValidPhotoUrl)
 	if err != nil {
```

Comment:
We are not actually handling the error.

Reply:
Done. Add a return of empty string when error occurs.

Reply:
So if we return an empty string, how would the caller handle it?

Reply:
The caller will just have an empty string for the photo url at the moment, and the place photo will still be missing. This PR mainly deals with the issue where Photo URL extraction returns `www.google.com`. 
In a separate PR, we can add the caller's handling of empty url. For example, we can use a default picture or icon.